frontend/src/concepts/pipelines/NoPipelineServer.tsx

```diff
@@ -1,28 +1,26 @@
 import * as React from 'react';
-import {
-  EmptyState,
-  EmptyStateIcon,
-  EmptyStateHeader,
-  EmptyStateFooter,
-} from '@patternfly/react-core';
-import { WrenchIcon } from '@patternfly/react-icons/dist/esm/icons/wrench-icon';
 import { CreatePipelineServerButton } from '~/concepts/pipelines/context';
+import EmptyDetailsView from '~/pages/projects/screens/detail/EmptyDetailsView';
```

Comment:
We want to avoid `concepts` importing `pages`. It happens today but we don't want to add to the situation. It's not enforced but we have rules defined that can warn us once we are ready to enforce it.